components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -779,33 +779,34 @@ void RewardsServiceImpl::TriggerOnWalletInitialized(int error_code) {
 
 void RewardsServiceImpl::TriggerOnWalletProperties(int error_code,
     std::unique_ptr<ledger::WalletInfo> wallet_info) {
-  std::unique_ptr<brave_rewards::WalletProperties> wallet_properties;
+  for (auto& observer : observers_) {
+    std::unique_ptr<brave_rewards::WalletProperties> wallet_properties;
 
-  if (!wallet_info) {
-    return;
-  }
+    if (!wallet_info) {
+      return;
+    }
 
-  wallet_properties.reset(new brave_rewards::WalletProperties);
-  wallet_properties->probi = wallet_info->probi_;
-  wallet_properties->balance = wallet_info->balance_;
-  wallet_properties->rates = wallet_info->rates_;
-  wallet_properties->parameters_choices = wallet_info->parameters_choices_;
-  wallet_properties->parameters_range = wallet_info->parameters_range_;
-  wallet_properties->parameters_days = wallet_info->parameters_days_;
+    wallet_properties.reset(new brave_rewards::WalletProperties);
+    wallet_properties->probi = wallet_info->probi_;
+    wallet_properties->balance = wallet_info->balance_;
+    wallet_properties->rates = wallet_info->rates_;
+    wallet_properties->parameters_choices = wallet_info->parameters_choices_;
+    wallet_properties->parameters_range = wallet_info->parameters_range_;
+    wallet_properties->parameters_days = wallet_info->parameters_days_;
 
-  for (size_t i = 0; i < wallet_info->grants_.size(); i ++) {
-    brave_rewards::Grant grant;
+    for (size_t i = 0; i < wallet_info->grants_.size(); i ++) {
+      brave_rewards::Grant grant;
 
-    grant.altcurrency = wallet_info->grants_[i].altcurrency;
-    grant.probi = wallet_info->grants_[i].probi;
-    grant.expiryTime = wallet_info->grants_[i].expiryTime;
+      grant.altcurrency = wallet_info->grants_[i].altcurrency;
+      grant.probi = wallet_info->grants_[i].probi;
+      grant.expiryTime = wallet_info->grants_[i].expiryTime;
 
-    wallet_properties->grants.push_back(grant);
-  }
+      wallet_properties->grants.push_back(grant);
+    }
 
-  // webui
-  for (auto& observer : observers_)
```

Comment:
This line was moved to the top of the function. Everything else is just indents.